docs/model_configuration.md

```diff
@@ -106,6 +106,15 @@ For models that do not support batching, or do not support batching in
 the specific ways described above, *max_batch_size* must be set to
 zero.
 
+When a model is using the auto-complete feature, a default maximum batch size may 
+may be set by using the `--backend-config=default-max-batch-size=<int>` command line switch.
+This allows all models which are capable of batching and which make use of auto-complete
+to have a non-zero maximum batch size, provided the backend which initializes the model
```

Comment:
to have a default maximum batch size. The backends that utilize this feature include: Tensorflow, TensorRT ...

Reply:
Decided that the phrase "The backends that utilize this feature include" would be redundant at this time as we go on to say that none of the Triton backends support this feature yet.